### DISCUSSION TITLE
How to receive mails with python?

### DISCUSSION THREAD

**Discussion by vadympop** [modoboa/modoboa#3071] [Help]
I need to receive all mails which come to modoboa to telegram chat using python tg bot

**Comment by H-A-L-I-S** [comment]
This is a more complicated project. You might start by getting familiar with the telegram bot api. This might take some time and by the way, you asked the question, may exceed your personal capabilities.

There could be ready to use solutions, which might make your project easier.

**Reply by vadympop** [reply]
I am well familiar with telegram bot api, I need to know how to receive messages from modoboa
